routing_common/vehicle_model.hpp

```diff
@@ -123,9 +128,15 @@ class VehicleModel : public IVehicleModel
     double const m_speedKMpH;
   };
 
+  struct RoadLimits
+  {
+    double m_speedKMpH;
```

Comment:
It's not expected to construct RoadLimits with default values so i deleted default constructor to make it clear. Is it ok?